### PR TITLE
[Big Tensor] Fix invalid test case for adaptive_log_softmax_with_loss

### PR DESCRIPTION
**Pull request by Difers**
https://github.com/PFCCLab/PaddleAPITest/pull/385
之前的PR遗漏了tester下的测例修改，在本PR做了补充。
当前的case其实并非完全合理输入，实际场景下该算子在nn.AdaptiveLogSoftmaxWithLoss中进行调用，当前的case在该场景下会非法，但基于该场景shape复杂且不好组合出大tensor情况的case；因此依旧保留当前case提供于大tensor测试
算子合理输入shape构造规则：

参数名 | shape/结构 | 说明
-- | -- | --
input | [batch_size, in_features] | 输入的特征张量，batch_size为批量大小，in_features为输入特征数
label | [batch_size] | 标签，通常是一维整数索引，长度为batch_size
head_weight | [in_features, head_size] | head部分的权重，head_size = shortlist_size + n_clusters
head_bias | [head_size] 或 None | head部分的偏置，只有当 head_bias=True 时有偏置
tail_weights | [[Tensor, Tensor], ...] | 长度为n_clusters，每个元素是2个Tensor:1. [in_features, hsz]2. [hsz, osz]；hsz = int(in_features // (div_value ** (i + 1)))；osz = cutoffs[i + 1] - cutoffs[i]
cutoffs | [cutoff_0, cutoff_1, ..., n_classes] | 长度为len(cutoffs)+1，最后一个为n_classes

### CODE REVIEW

File: tester/api_config/8_big_tensor/big_tensor_1_8.txt [hunk 1]
```diff
@@ -29598,9 +29598,10 @@ paddle.nn.functional.adaptive_avg_pool3d(x=Tensor([2, 3, 8, 32, 2796203],"float1
...
 paddle.nn.functional.adaptive_avg_pool3d(x=Tensor([3104356, 3, 5, 7, 7],"float32"), output_size=list[None,3,None,], )
 paddle.nn.functional.adaptive_avg_pool3d(x=Tensor([92843, 3, 8, 32, 32],"float32"), output_size=list[3,3,3,], )
-paddle.nn.functional.adaptive_log_softmax_with_loss(Tensor([142606337, 16],"float32"), Tensor([128],"int64"), Tensor([16, 8],"float32"), list[list[Tensor([16, 8],"float32"),Tensor([8, 5],"float32"),],list[Tensor([16, 4],"float32"),Tensor([4, 5],"float32"),],list[Tensor([16, 2],"float32"),Tensor([2, 5],"float32"),],], list[5,10,15,20,], None, )
-paddle.nn.functional.adaptive_log_softmax_with_loss(Tensor([285212673, 8],"float32"), Tensor([8],"int64"), Tensor([8, 3],"float32"), list[list[Tensor([8, 4],"float32"),Tensor([4, 2],"float32"),],], list[2,4,], None, )
+paddle.nn.functional.adaptive_log_softmax_with_loss(Tensor([132606337, 16],"float32"), Tensor([132606337],"int64"), Tensor([16, 8],"float32"), list[list[Tensor([16, 8],"float32"),Tensor([8, 5],"float32"),],list[Tensor([16, 4],"float32"),Tensor([4, 5],"float32"),],list[Tensor([16, 2],"float32"),Tensor([2, 5],"float32"),],], list[5,10,15,20,], None, )
```
Comment (cangtianhuang) [inline tester/api_config/8_big_tensor/big_tensor_1_8.txt:29601]: 这个测试用例的 numel 没有超过 int32

File: tester/api_config/8_big_tensor/big_tensor_paddleonly.txt [hunk 2]
```diff
@@ -684,8 +684,9 @@ paddle.nn.functional.adaptive_avg_pool3d(x=Tensor([3104356, 3, 5, 7, 7],"float32
 paddle.nn.functional.adaptive_avg_pool3d(x=Tensor([3104356, 3, 5, 7, 7],"float32"), output_size=list[3,3,3,], )
 paddle.nn.functional.adaptive_avg_pool3d(x=Tensor([3104356, 3, 5, 7, 7],"float32"), output_size=list[3,3,3,], data_format="NDHWC", )
 paddle.nn.functional.adaptive_log_softmax_with_loss(Tensor([285212673, 8],"float32"), Tensor([285212673],"int64"), Tensor([8, 3],"float32"), list[list[Tensor([8, 4],"float32"),Tensor([4, 2],"float32"),],], list[2,4,], None, )
-paddle.nn.functional.adaptive_log_softmax_with_loss(Tensor([285212673, 8],"float32"), Tensor([4],"int64"), Tensor([8, 3],"float32"), list[list[Tensor([8, 4],"float32"),Tensor([4, 2],"float32"),],], list[2,4,], None, )
+paddle.nn.functional.adaptive_log_softmax_with_loss(Tensor([132606337, 16],"float32"), Tensor([132606337],"int64"), Tensor([16, 8],"float32"), list[list[Tensor([16, 8],"float32"),Tensor([8, 5],"float32"),],list[Tensor([16, 4],"float32"),Tensor([4, 5],"float32"),],list[Tensor([16, 2],"float32"),Tensor([2, 5],"float32"),],], list[5,10,15,20,], None, )
```
Comment (cangtianhuang) [inline tester/api_config/8_big_tensor/big_tensor_paddleonly.txt:687]: 同上

**Review by cangtianhuang — CHANGES_REQUESTED** [review]

**Review by cangtianhuang — APPROVED** [review]
LGTM
